### ISSUE TITLE
NPE when searching a bird nest

### ISSUE THREAD

**Issue by emerald000** [dillydill123/inventory-setups#64]
When you click on a bird nest to "search" and receive the seeds, the plugin throws an NPE:

```
WARN  n.runelite.client.eventbus.EventBus - Uncaught exception in event subscriber
java.lang.NullPointerException: null
	at inventorysetups.InventorySetupsPlugin.onMenuOptionClicked(InventorySetupsPlugin.java:549)
	at net.runelite.client.eventbus.EventBus$Subscriber.invoke(EventBus.java:73)
	at net.runelite.client.eventbus.EventBus.post(EventBus.java:222)
	at net.runelite.client.callback.Hooks.post(Hooks.java:165)
	at hn.hm(hn.java:31423)
	at client.cr(client.java:3407)
	at client.ah(client.java:1587)
	at ba.x(ba.java:335)
	at ba.run(ba.java:314)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by dillydill123** [comment]
I think this is occurring for anything with "Search" because there is some code to make the search function of banks work with the filtering. I am not checking to see if the bank is open (widget is null) before trying to  use it. If you don't have a setup selected, I think you will not get the NPE.

Nice find! I will fix this.

**Comment by dillydill123** [comment]
Were you searching the nest in a specific area? I can't seem to get reproduce the bug. The widget is not null I will still add this check in though.

**Comment by emerald000** [comment]
I was right in front of the south-west bird house.

**Comment by dillydill123** [comment]
Ahh I got it. If you don't visit a bank before searching a nest, the widget will be null and cause the NPE.

**Comment by dillydill123** [comment]
Fixed in 1.8.2
